src/components/Auth/AuthButton.js

```diff
@@ -33,14 +45,48 @@ const AuthButton = ({ first_name }) => {
       }
       {userIDCookie && (
         <>
-          {first_name && (
-            <label style={{ color: "#0c3c78", fontSize: "1.1rem" }}>
-              Hello {first_name}!
-            </label>
-          )}
-          <Button onClick={() => logoutWithRedirect()} color="secondary">
-            Sign out
+          {" "}
+          <Button
+            aria-controls="simple-menu"
+            aria-haspopup="true"
+            onClick={handleClick}
```

Comment:
How is this working if we're not passing in an `e` argument?

Reply:
MUI docs show no argument is needed.